doc/build-unix.md

```diff
@@ -40,7 +40,7 @@ make install # optional
 
 Or, to keep the source directory clean:
 ```bash
-./autogen.sh && mkdir build && ../configure && make
+./autogen.sh && mkdir build && cd build && ../configure && make
 ```
 
 This will build gridcoinresearch (Qt client) as well if the dependencies are met.
```

Comment:
I really don't like the chained commands. It makes it difficult for novices to be able to tell where the failure is, then they come on and say my build failed, and you ask where, and they have no idea. One command at a time on the command line. Also there should be a treatment of -j[number of jobs], because most people have enough memory to run several jobs at once and it drastically speeds up the compiler. A good rule of thumb is the lesser of the number of CPU's or the # of GB of memory in the machine, whichever is less.

Reply:
Unchained, and added -j with core count and a comment saying it's changable. Is that ok?

Reply:
Perfect! :)